aws/service.go

```diff
@@ -1940,5 +1958,11 @@ func (d ConnectionErrRetryer) RetryRules(r *request.Request) time.Duration {
 	// Creates a new exponential backoff using the starting value of
 	// minDelay and (minDelay * 3^retrycount) * jitter on each failure
 	// as example (23.25ms, 63ms, 238.5ms, 607.4ms, 2s, 5.22s, 20.31s...) up to max.
-	return time.Duration(int(float64(int(minDelay.Nanoseconds())*int(math.Pow(3, float64(retryCount)))) * jitter))
+	// Maximum delay should not be more than 5 min
+	maxDelayTime := time.Duration(int(float64(int(minDelay.Nanoseconds())*int(math.Pow(3, float64(retryCount)))) * jitter))
+	if maxDelayTime > time.Duration(5*time.Minute) {
```

Comment:
Did you do any testing with 3 minutes (our previous max time before)? Is there any difference in performance, or do the subsequent API calls that we retry also succeed when the max is 3 minutes?